project/ProjectPlugin.scala

```diff
@@ -158,37 +151,21 @@ object ProjectPlugin extends AutoPlugin {
 
   }
 
-  import autoImport._
-
   override def projectSettings: Seq[Def.Setting[_]] =
-    Seq(
-      resolvers += Resolver.bintrayRepo("beyondthelines", "maven"),
-      orgAfterCISuccessTaskListSetting := List(
-        depUpdateDependencyIssues.asRunnableItem,
-        orgPublishReleaseTask
-          .asRunnableItem(allModules = true, aggregated = false, crossScalaVersions = true),
-        orgUpdateDocFiles.asRunnableItem
-      ),
+    scalaMetaSettings ++ sharedReleaseProcess ++ warnUnusedImport ++ Seq(
+      libraryDependencies ++= commonDeps :+ %("slf4j-nop") % Test,
```

Comment:
Adding `slf4j-nop` to tests fixes the annoying red warnings about missing `slf4j` logger implementations.